src/Serializer/Normalizer/PhoneNumberNormalizer.php

```diff
@@ -85,11 +85,11 @@ public function supportsNormalization($data, $format = null)
      */
     public function denormalize($data, $class, $format = null, array $context = [])
     {
-        if (null === $data) {
-            return;
-        }
-
         try {
+            if (!(bool) $data) {
+                return null;
+            }
```

Comment:
It's useless in the try, and still bad for the business here.

Reply:
Yes I'll put it off the try

Reply:
I mean, this `if (!(bool) $data)` is not a good change imo.